graphql/resolvers.go

```diff
@@ -1599,8 +1599,13 @@ func (r *queryResolver) Projects(ctx context.Context) ([]*GroupedProjects, error
 	if err != nil {
 		return nil, ResourceNotFound.Send(ctx, err.Error())
 	}
-
-	groupedProjects, err := GroupProjects(allProjs, false)
+	enabledProj := []model.ProjectRef{}
```

Comment:
nit: can we call this enabledProjects since it's a list (and more consistent with groupedProjects)

Reply:
Also if you think this change will confuse someone in the future, I would add a comment saying we need to iterate over the merged projects which is why we don't query directly (or something)